xds/googledirectpath/googlec2p.go

```diff
@@ -120,7 +120,8 @@ func (c2pResolverBuilder) Build(t resolver.Target, cc resolver.ClientConn, opts
 		ClientDefaultListenerResourceNameTemplate: "%s",
 		Authorities: map[string]*bootstrap.Authority{
 			c2pAuthority: {
-				XDSServer: serverConfig,
```

Comment:
nit: can we add a TODO comment to explain how this only needed b/c it doesn't use the parsing logic in bootstrap.go which would normally set this default?

(otherwise it's a little confusing how this looks different from java/c++)

Reply:
This is all changing in https://github.com/grpc/grpc-go/pull/7040. I wanted to fix the behavior as part of this PR and try to make it more obvious in the other PR